.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "semi": false,
   "singleQuote": true,
-  "trailingComma": "all"
+  "trailingComma": "all",
+  "endOfLine": "lf"
```

Comment:
endOfLine lf is the prettier default since v2

https://prettier.io/docs/en/options.html#end-of-line